### ISSUE TITLE
Multiple nuclei or cytoplasm markers

### ISSUE THREAD

**Issue by ellispatrick** [SydneyBioX/simpleSeg#1]
It would be good to be able to use multiple nuclei markers, ie nucleus = c("HH3", "nuc") or cellBody = c("...", "...", "...").

This would probably just need to be scaled in some way and then averaged.

**Comment by anic0077** [comment]
Done, though I have left out the scaling for now

**Comment by goterm** [comment]
Hi,

I bumped into simplSeg functions and it's being great but I cannot use cytoplasmic markers in simpleSeg(). Using the dummy data provided with the package (https://bioconductor.org/packages/devel/bioc/vignettes/simpleSeg/inst/doc/simpleSeg.html) I get:

masks <- simpleSeg::simpleSeg(images,
                              nucleus = "HH3",
                              cellBody = "CD45",
                              transform = "sqrt")
Error in simpleSeg::simpleSeg(images, nucleus = "HH3", cellBody = "CD45",  : 
  Invalid method of cytoplasm identification: 'CD45'. Must be none, dilate, discModel.

I have also tried with cellBody = dimnames(images[[1]])[[3]][10], but the error persist.
any help would be appreciated,
thanks in advance
